website/docs/guides/0.8.2-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,59 @@
+## Issue [INTMDB-175](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/398)
+### resources created with the affected versions ***v0.8.0/v0.8.1***
+
+The following resources were affected, due to a schema update,
+to achieve consistency for multiple providers (AWS) ()[]
+
+```
+mongodbatlas_privatelink_endpoint
+mongodbatlas_privatelink_endpoint_service
+```
+
+### Known issue from v0.8.0 to v0.8.2
+
+If the resources were created from v0.8.0, there was no error in the creation.
```

Comment:
If the resources were created with v0.8.0, there was no error in the creation of the resource in Atlas.